Source/MOLXPCConnection/MOLXPCConnection.h

```diff
@@ -94,6 +96,11 @@
  */
 - (void)resume;
 
+/**
+  Enables or disables the connection from unprivileged users. (server)
+*/
+- (void)allowUnprivilegedClients:(BOOL)enable;
```

Comment:
I think this property can be removed and the presence of unprivilegedExportedInterface can be used to infer this. I'll elaborate more below.